catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/suggestion/UtmUpsCoordinateProcessor.java

```diff
@@ -76,42 +79,178 @@ public UtmUpsCoordinateProcessor(CoordinateSystemTranslator translator) {
+      return Stream.of(
+              parseUtmUpsString(removeLatBand(utmUpsCoord) + " N"), parseUtmUpsString(utmUpsCoord))
+          .filter(Objects::nonNull)
+          .map(this::suggestion)
+          .collect(toList());
+    } else if (latBand.equals('S')) {
+      // Similarly, if the coordinate has an S lat band, include the southern hemisphere
+      // suggestion for convenience
+      return Stream.of(
+              parseUtmUpsString(removeLatBand(utmUpsCoord) + " S"), parseUtmUpsString(utmUpsCoord))
+          .filter(Objects::nonNull)
+          .map(this::suggestion)
+          .collect(toList());
+    } else {
+      return Stream.of(parseUtmUpsString(utmUpsCoord))
+          .filter(Objects::nonNull)
+          .map(this::suggestion)
+          .collect(toList());
     }
+  }
+
+  private LiteralSuggestion suggestion(UtmUpsCoordinate utmUps) {
     return new LiteralSuggestion(
         LITERAL_SUGGESTION_ID,
-        nameBuilder.toString(),
+        makeSuggestionText(utmUps),
+        Collections.singletonList(latLonFromUtmUtps(utmUps)));
+  }
+
+  /**
+   * Generates one {@link LiteralSuggestion} by combining the latitude/longitude values of each
+   * UTM/UPS coordinate in the {@code utmUpsCoords} list. This method is used to generate a
+   * suggestion when the user enters multiple UTM/UPS coordinates. No additional suggestions are
+   * made for each point for simplicity since the map will pan to the extent or bounding box of all
+   * the points. As a result, the N/S characters of each coordinate are always treated as latitude
+   * bands and a coordinate is considered invalid if a latitude band is not included. Example: "12S
+   * 241451mE 4101052mN 13R 37090mE 63394439mN" -> "UTM/UPS: [ 12S 241451mE 4101052mN ] [ 13R
+   * 37090mE 63394439mN ]"
+   *
+   * @param utmUpsCoords a list of strings matching the {@code PATTERN_UTM_OR_UPS_COORDINATE}
+   *     pattern, created from the user-provided gazetteer query text.
+   * @return a singleton list with one {@link LiteralSuggestion} with the combined list of
+   *     latitude/longitude values.
+   */
+  private LiteralSuggestion suggestionForMultiplePoints(final List<String> utmUpsCoords) {
+    List<UtmUpsCoordinate> utmUpsList =
         utmUpsCoords
             .stream()
-            .map(this::toLatLon)
+            .map(this::parseUtmUpsString)
             .filter(Objects::nonNull)
-            .map(d -> new LatLon(d.getLat(), d.getLon()))
-            .collect(Collectors.toList()));
+            .collect(toList());
+    List<LatLon> latLonList =
+        utmUpsList.stream().map(this::latLonFromUtmUtps).filter(Objects::nonNull).collect(toList());
+    String suggestionText = makeSuggestionText(utmUpsList);
+
+    return new LiteralSuggestion(LITERAL_SUGGESTION_ID, suggestionText, latLonList);
+  }
+
+  private static String makeSuggestionText(UtmUpsCoordinate utmUps) {
+    final StringBuilder nameBuilder = new StringBuilder();
+    if (utmUps.isUTM()) {
```

Comment:
❗️ Same here. This isn't exactly what you want.

Reply:
✅ After further analysis we could not reproduce the problem. Turns out the semantics of `UtmUpsCoordinate` are different from the other coordinate impls. This is fine.